test/param.spec.js

```diff
@@ -433,7 +442,7 @@ describe('Testing Params', () => {
   });
 
   it('Testing Fields Parameter (query)', () => {
-    const param = api.FieldsParam('param', ['id', 'user.id', 'user.name']);
+    const param = api.FieldsParam('param', { paths: ['id', 'user.id', 'user.name'] });
```

Comment:
Urg... so this is breaking everywhere? Preference is to have it as an arg and now a kwarg... we can redesign that when we rework this whole library.